NEWS.md

```diff
@@ -9,6 +9,7 @@
 ## formatters 0.3.3.3
  * length-zero values of `na_str` are now silently interpreted as the default value (`"NA"`),
    as are any missing values in `na_str` of length > 0.
+ * adding wrapping of titles, subtitles, and footers.
```

Comment:
This should be under a new 0.3.3.6 heading, not under 0.3.3.3 (since it is being added after 0.3.3.5 hit main)